victoria_metrics.go

```diff
@@ -11,57 +11,117 @@ func (b *Builder) GetOrCreateCounter() *metrics.Counter {
 	return metrics.GetOrCreateCounter(b.String())
 }
 
+// GetOrCreateCounterInSet calls [set.GetOrCreateCounter] using the Builder's accumulated string as argument.
```

Comment:
I believe that for pkg.go.dev to properly parse the godoc, you need to have the whole path to the method with the types properly formatted, so instead of :
`[set.GetOrCreateCounter]` 
here, you would need :
`[metrics.Set.GetOrCreateCounter]`.

Reply:
didn't realise this was interpreted by godoc, I'll change it.